### PR TITLE
Update docs development and tools sections

### PR DESCRIPTION
**Pull request by LaakkonenJussi**
Fix links, change example locations, add --no-gpg-checks and single repo refreshing.

### CODE REVIEW

**Review by jovirkku — APPROVED** [review]
Looks good and clear to me. The additional commands are explained well.

I have one question on line 84 (the same on 46 and 53).

File: Services/Development/Webhooks/README.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ You can just branch the package - you'll get a webhook automatically. You can th
 
 Branching is useful as it will ensure the same build repositories are used:
 
-1.  Go to the web page for a project you'd like to branch (e.g. <https://build.sailfishos.org/package/show/mer:core/curl>)
+1.  Go to the web page for a project you'd like to branch (e.g. <https://build.sailfishos.org/package/show/sailfishos:chum/wget>)
 2.  Click on "Branch package" link
 3.  Click "Ok" (or check out more options)
 
-You can then edit the branched `_service` file and change "repourl" from <https://github.com/sailfishos/curl.git> to e.g. <https://github.com/my-forks/curl.git>. Branch can also be changed to e.g. *myfeaturebranch*.
+You can then edit the branched `_service` file and change "repourl" from <https://github.com/sailfishos/wget.git> to e.g. <https://github.com/my-forks/wget.git>. Branch can also be changed to e.g. *myfeaturebranch*.
```
[the remark this answers is not in the document]
Comment (LaakkonenJussi) [reply Services/Development/Webhooks/README.md:85]: Ah, right, we didn't have that `curl` on the build.sailfishos.org. And apparently we don't have wget anymore, heh. Need to pick up some other repo here that exists on both build.sailfishos.org (under CHUM) and as a git repo.

[the remark this answers is not in the document]
Comment (LaakkonenJussi) [reply Services/Development/Webhooks/README.md:85]: Not sure though which package would be a good example here...

[the remark this answers is not in the document]
Comment (LaakkonenJussi) [reply Services/Development/Webhooks/README.md:85]: I wonder if that pcre2 is a good example. I guess the point here is to bring an example we have as a git repo and the same is also buildable in the community OBS? The curl-link there was broken so tried to find a replacement here. Any comments @rainemak ?

**Review by rainemak — APPROVED** [review]

**Pull request merged**